dendrite-config.yaml

```diff
@@ -300,6 +300,11 @@ sync_api:
     max_idle_conns: 2
     conn_max_lifetime: -1
 
+  # This option controls which HTTP header to inspect to find the real remote IP
+  # address of the client. This is likely required if Dendrite is running behind
+  # a reverse proxy server.
+  # real_ip_header: X-Real-IP
```

Comment:
We presumably want the same config option on `client_api`?

Reply:
Nothing in the client API right now actually updates the last seen, only sync.

Reply:
We could put the option into the global section and then other components could pull it in if needed?

Reply:
I think there's only 3 client-facing components: media/sync/client API. Probably shouldn't pollute global.